frontend/js/models/scale.js

```diff
@@ -82,6 +80,18 @@ define(
                 return undefined;
             },
 
+            /**
+             * Parse the attribute list passed to the model
+             * @param {object} data Object literal containing the model attribute to parse.
+             * @return {object} The object literal with the list of parsed model attribute.
+             */
+            parse: function (data) {
+                if (data.scaleValues) {
+                    data.scaleValues = new ScaleValues(data.scaleValues, { scale: this });
```

Comment:
This seems sketchy somehow, might be worth explaining in a comment why this is necessary.